jwst/lib/set_telescope_pointing.py

```diff
@@ -304,14 +308,15 @@ def update_wcs_from_fgs_guiding(model, default_pa_v3=0.0, default_vparity=1):
 
     # Get position angle
     try:
-        pa = model.meta.wcsinfo.pa_v3
+        pa = model.meta.wcsinfo.roll_ref if model.meta.wcsinfo.roll_ref is not None else default_roll_ref
     except AttributeError:
         logger.warning(
-            'Keyword `PA_V3` not found. Using {} as default value'.format(
-                default_pa_v3
+            'Keyword `ROLL_REF` not found. Using {} as default value'.format(
+                default_roll_ref
             )
         )
-        pa = default_pa_v3
+        pa = default_roll_ref
```

Comment:
Are these doing the same thing?  It looks like `pa` is getting populated regardless of the `except` block, then also getting populated in the `except` block.

Also, if `pa` is the position angle at the aperture reference point, i.e `roll_ref`, the variable should just be called `roll_ref` to avoid confusion.

Reply:
I was running into a case where `model.meta.wcsinfo.roll_ref` was returning `None` rather than raising an attribute error, which then caused errors elsewhere

Reply:
Yeah, it should always return `None` if it is not populated, as it is in the schema for all datamodels.  I.e. it should never raise `AttributeError`, though I suppose one can't be too careful?

Reply:
Hmmm, well the reason that I implemented it in this way is because I was trying to stick with what was previously done with the other values (in the try/except blocks). Perhaps those should be updated as well?